locales/ka-GE/src/comments.ts

```diff
@@ -2,186 +2,188 @@ import {removeUndefinedLocaleResources} from 'sanity'
   /** When a users' name cannot be resolved, fall back to this name */
-  'reactions.user-list.unknown-user-fallback-name': undefined, // 'Unknown user'
+  'reactions.user-list.unknown-user-fallback-name': 'უცნობი მომხმარებელი',
   /**
    * When showing list of users who have reacted, replaces your own name with "you", eg
    * "Donna, you, James, and Tyler reacted with 👍". A different key (`_leading` suffix)
    * is used when you are the first to react, eg "You, Donna and Tyler reacted with 👍".
    * Use `{{name}}` if you want to instead use the current users' actual name.
    */
-  'reactions.user-list.you': undefined, // 'you'
+  'reactions.user-list.you': 'შენ',
   /**
    * When showing list of users who have reacted, replaces your own name with "You", eg
    * "You, Donna, James, and Tyler reacted with 👍". A different key (`_leading` suffix)
    * is used when you are not the first to react, eg "Donna, you, James and Tyler reacted with 👍".
    * Use `{{name}}` if you want to instead use the current users' actual name.
    */
-  'reactions.user-list.you_leading': undefined, // 'You'
+  'reactions.user-list.you_leading': 'შენ',
   /**
    * The text shown for the tooltip that appears when hovering over the reaction count, eg
    * "Donna, James, and Tyler Reacted with 👍". Three components are available for use:
    * - `<UserList/>` - the list of names of users who have reacted, using the locales list format
    * - `<Text>` - should be wrapped around the text describing the action ("reacted with")
    * - `<ReactionName/>` - the name of the reaction emoji, eg ":heavy_plus_sign:"
    */
-  'reactions.users-reacted-with-reaction': undefined, // '<UserList/> <Text>reacted with</Text> <ReactionName/>'
+  'reactions.users-reacted-with-reaction': '<UserList/> <Text>რეაგირებულია</Text> <ReactionName/>',
 
   /** Status filter: The short title describing filtering on open (non-resolved) comments */
-  'status-filter.status-open': undefined, // 'Open'
+  'status-filter.status-open': 'ღია',
   /** Status filter: The full text for describing filtering on open (non-resolved) comments */
-  'status-filter.status-open-full': undefined, // 'Open comments'
+  'status-filter.status-open-full': 'ღია კომენტარები',
   /** Status filter: The short title describing filtering on resolved comments */
-  'status-filter.status-resolved': undefined, // 'Resolved'
+  'status-filter.status-resolved': 'გადაწყდებული',
   /** Status filter: The full text for describing filtering on resolved comments */
-  'status-filter.status-resolved-full': undefined, // 'Resolved comments'
+  'status-filter.status-resolved-full': 'გადაწყდებული კომენტარები',
   /** Status filter: The full text for describing filtering on resolved comments and is upsell mode */
-  'status-filter.status-resolved-full-upsell': undefined, // 'Upgrade to see resolved comments'
+  'status-filter.status-resolved-full-upsell': 'განახლება გადაწყდებული კომენტარების სანახავად',
 })
```

Comment:
**here is the correct verisons **

'copy-link-success-message': 'ბმული დაკოპებულია',
'delete-comment.title': 'ნამდვილად გსურთ კომენტარის წაშლა?',
'delete-thread.title':'ნამდვილად გსურთ კომენტარის წაშლა?',
'feature-feedback.link': 'გაგვიზიარეთ თქვენი შეფასება',
'list-status.empty-state-resolved-title': 'ჯერ არ არის გაცხრილული კომენტარები',
'status-filter.status-resolved-full': 'გაცხრილული კომენტარები',
'status-filter.status-open': 'გახსნა',
'mentions.unauthorized-user': 'უფლებამოსილი',
'status-filter.status-resolved': 'გაცხრილული',
'list-item.layout-failed-sent': 'გაგზავნა ვერ მოხერხდა.',